crates/meroctl/src/cli/context/event.rs

```diff
@@ -0,0 +1,109 @@
+use std::process::Command;
```

Comment:
use tokio's [`Command`](https://docs.rs/tokio/latest/tokio/process/struct.Command.html) instead of `std`s